### DISCUSSION TITLE
Snowflake novel assitant

### DISCUSSION THREAD

**Discussion by ghost** [vkbo/novelWriter#853] [Ideas]
Do you have plans for a snowflake novel assistant like Manuskript?

**Comment by vkbo** [comment]
I have thought about it. My work flow is based on the snowflake method, so there are features in novelWriter that makes this method easier to use. That is why the Synopsis feature and Split tool exist.

The way I apply the method is that I start with a coarse plot summary in a single document, and break it down into acts and plot stages, which are then broken down into the key scenes with a synopsis of the 10 key plot points. Sometimes, depending on the plot complexity, I will break it down further into more key scenes using the synopsis feature. All of this is done in one file. I then split that file up into the respective scene files by using the Split tool set to split on level 3 headings.

Since the split tool doesn't delete the original file it splits, that file can be kept for reference, and I can continue writing in the newly created scene files.

Of course, this is a manual application of the snowflake method. Since novelWriter doesn't have forms for content and meta data like Manuskirpt does, there is no direct way of making an assistant as the program exists now. It can of course be added and the data saved as a JSON file, so it is possible.

I can also make a new template project that is populated with files prepared for using the snowflake method. That would be more in line with how novelWriter currently works. I think most people who've made a few projects anyway adapt the method for their own use, if they use it at all. I consider it more a learning tool or a guide than a strict method to follow in detail, so I've personally never cared much for the step by step forms that guide you through it.

I'm happy to accommodate this in some form as I quite like the principle of the method. Would a snowflake template be a good solution? I'm anyway planning to improve the template method so that the user can generate their own project templates as proposed in #841.

**Reply by ghost** [reply]
A template is fine. Sure!

The snowflake method is very appealing for beginners. The step-by-step approach is a way to keep the wannabe writer moving forward. I don't use it all the time, but it's something cool to have. However, this is not a killer feature. Whatever works better for novelWriter's workflow would be great.

**Reply by vkbo** [reply]
I agree, it's great for beginners. The first steps of the snowflake method is also a great way to get started on a new idea. I like to plan my plots in detail in advance, so I always use it in one way or another.

I think a few pre-generated files with the steps as outlined in the method, with some comments explaining the steps, as well as a character template, would be sufficient. A single note outlining the 10 steps of the method can also be added.

I'll reference this discussion in the template feature issue. I'll add such a template when I write the template code.

**Reply by ghost** [reply]
Is this still in your plans?

**Reply by vkbo** [reply]
It is. The feature is queued up in #841. I was originally planning to including it in the upcoming release, but the backlog is rather large compared to how much available time I have, so it may take a while still.
